buildspec.yml

```diff
@@ -18,7 +18,6 @@ phases:
         commands:
             - yarn generate
             - yarn prisma generate
-            - yarn prisma migrate deploy
```

Comment:
I think we should do the deploy during EBS init rather than during the build step (this is still TODO), it seems wrong for a build step to be modifying external services. What do you think?